app/code/Magento/CatalogGraphQl/etc/schema.graphqls

```diff
@@ -10,6 +10,10 @@ type Query {
         sort: ProductAttributeSortInput @doc(description: "Specifies which attributes to sort on, and whether to return the results in ascending or descending order.")
     ): Products
     @resolver(class: "Magento\\CatalogGraphQl\\Model\\Resolver\\Products") @doc(description: "The products query searches for products that match the criteria specified in the search and filter attributes.") @cache(cacheIdentity: "Magento\\CatalogGraphQl\\Model\\Resolver\\Product\\Identity")
+    product (
+        id: Int @doc(description: "Id of the product.")
```

Comment:
Optional, but don't really need this comment - the schema here is plenty declarative. If it's not, we can call the field `productID`